### PR TITLE
NMS-11810: Report documentation

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [X] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-11810

### CODE REVIEW

File: docs/modules/operation/pages/database-reports/templates/calendar.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+This report is generated directly from the {page-component-title} Java code and has no optional parameters for selecting a subset of the network devices.
+The report contains the same information as the "Default Classic Report" with the exception that the information in the second and third tables presents the data in the format of a calendar month.
+
+The period is the current month to the last full day before the day the report was generated (in other words, yesterday).
+All the data is presented in tables.
+
+The first table contains a table of the monthly availability percentage for all services {page-component-title} monitored for the past year.
+It also lists the number of nodes that had outages, the number of interfaces, and the number of services.
+// QUESTION: Not sure if this last set of information show the maximum or current number of interfaces and services.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/database-reports/templates/calendar.adoc:15]: @mmahacek did something go wrong? I thought that Andrew had removed all the questions.

**Review by Bonrob2 — COMMENTED** [review]
There might be an issue here, as I thought that Andrew had removed all of the questions from the .adoc files, and I don't remember seeing them in the last review I did.

**Comment by mmahacek** [comment]
I don't think they were fully removed after the last approval.

**Comment by Bonrob2** [comment]
> 

> I don't think they were fully removed after the last approval.

You're right, I just went back and checked. We can keep them in for now. It's a good reminder that there are some outstanding issues.

**Comment by Bonrob2** [comment]
hey @mmahacek , I'm unable to request that I re-review this, but if you do I can approve the merge.

**Pull request merged**